social/apps/django_app/default/fields.py

```diff
@@ -11,7 +11,7 @@
     from django.utils.encoding import smart_text
 
 
-class JSONField(six.with_metaclass(models.SubfieldBase, models.TextField)):
+class JSONField(models.TextField):
```

Comment:
@melvinsoft Thanks! Answers my question on that and I learned a little something in the process. So given what I said about my knowledge here in my previous comment, LGTM!